go.mod

```diff
@@ -13,31 +13,31 @@ replace (
 	google.golang.org/grpc => google.golang.org/grpc v1.29.1 // etcd depends on google.golang.org/grpc/naming which was removed in grpc v1.30.0
 	helm.sh/helm/v3 => github.com/rancher/helm/v3 v3.3.0-rancher1
 
-	k8s.io/api => k8s.io/api v0.20.0
-	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.20.0
-	k8s.io/apimachinery => k8s.io/apimachinery v0.20.0
-	k8s.io/apiserver => k8s.io/apiserver v0.20.0
-	k8s.io/cli-runtime => k8s.io/cli-runtime v0.20.0
+	k8s.io/api => k8s.io/api v0.20.6
```

Comment:
Did you look to see if there are any breaking api changes between 0.20.0 and 0.20.6 ?

Reply:
There are no listed breaking changes between these two versions.